tools/dep_whitelist/WorkspaceDepsWhitelistTest.java

```diff
@@ -38,6 +38,12 @@
 
 import org.apache.commons.lang3.StringUtils;
 
+/*
+  Test verifies whether all external dependencies
```

Comment:
Please add:
To simplify the test, we make the following assumptions:
- '://' appears only once in each line
- '://' always appears in the same line as '='